### ISSUE TITLE
Unique entities that should be shared among all scenario factsheets

### ISSUE THREAD

**Issue by adelmemariani** [OpenEnergyPlatform/oekg-builder#10]
There are entities in the OEKG that should be shared among all factsheets. For example, different factsheets, cover different sets of sectors. Therefore, the layout should provide a set of options to select from like below:

![Screenshot 2022-08-24 at 20 27 33](https://user-images.githubusercontent.com/7637364/186496723-9bdf937a-3193-4a98-9acc-f92c7f82b4c9.png)

Each of the checkboxes refers to an entity in the knowledge graph and when for example two study factsheets select 'Industry sector', they both refer to the same entity in the knowledge graph.  
  
But some other entities belong to a factsheet and need not be shared. For example, suppose every study has its own 'analysis scope'. Therefore, we can have a layout like the below for adding labels or other properties.

![Screenshot 2022-08-24 at 20 28 06](https://user-images.githubusercontent.com/7637364/186501297-93ae72d6-d4d8-429b-a09f-489140b8f1ba.png)

what is your viewpoint?

**Comment by MGlauer** [comment]
IMO a list of checkboxes does not suffice for two reasons:
1. The list of possible options might be long (e.g. for authors)
2. There is no way to add new instances

**Comment by tillmo** [comment]
Indeed. Here is some design proposal. Consider authors. There is an input box where you can enter an existing author, or create a new one. And you should be able to add new input boxes (eg. by clicking on a +) for the case of multiple authors. A simple version would work with select boxes, showing the existing authors plus an option "new author". In a more sophisticated version, the input box would have auto completion showing a list of matching existing authors to choose from. If the author is not in the system yet, then (when pressing enter after having entered the name of the author) you will be able to create a new author.

**Comment by MGlauer** [comment]
Maybe something like this?
![Screenshot from 2022-08-25 16-53-31](https://user-images.githubusercontent.com/7835111/186698505-013965d4-76a4-468b-a46d-312553a6a01a.png)

**Comment by tillmo** [comment]
yes

**Comment by adelmemariani** [comment]
I added an _'autocomplete'_  dropdown with a textbox and checkboxes. When the user checks some options, they will be shown below the dropdown. If the user writes some characters in the textbox of the dropdown, the options are filtered according to the matched terms.

 I also think we do not need to show the hierarchy on the left side. Because we can make the hierarchy in the background when we are about to make the RDF graph. I think the user is only interested in seeing forms to fill out. Therefore, I removed the _'tree view'_ on the left and added _'tabs'_ on top.

Selections:
![Screenshot 2022-08-29 at 16 59 49](https://user-images.githubusercontent.com/7637364/187232247-1fcc9fab-ba96-4875-8e83-d7f6829007b9.png)

Selected ones:
![Screenshot 2022-08-29 at 17 01 04](https://user-images.githubusercontent.com/7637364/187232120-f755d952-9a49-4462-bd48-d85b3fc27fe4.png)

Autocomplete:
![Screenshot 2022-08-29 at 17 14 18](https://user-images.githubusercontent.com/7637364/187235329-37530741-3637-4b6b-b7fc-45d0019d8912.png)

I will try to add a button for being able to add _new items_, in case there is no matching term.
What do you think about the changes?
